File: java/Animal.java
import java.util.ArrayList;
import java.util.Set;

public class Animal {
     int id;  
     String nombre; 
     ArrayList<String> raza;
     Double peso;
     int talla ; 
                  //atributos 
                  //obligatorios
     public Animal(String nombre ){
         this.nombre= nombre;
     }
     
     void printDataAnimal()
     {
        System.out.println("Identificador  : " + id + "\n"+ "Nombre   : " + nombre + "\n"+ "Raza    : " + raza+ "\n"+"Peso   : "+peso +"\n"+ "Talla : " + talla);
     }
}

File: python/animal.py
from tokenize import Double
import pkg_resources


class Animal :
    id      = int 
    nombre  = str
    raza    = []
    peso    = float
    talla   = int
    
    def __init__(self, nombre,peso):
        self.nombre = nombre
        self.peso = peso 
        
        
